server.js

```diff
@@ -64,4 +65,14 @@ async function handlePostBooks(req, res) {
   
 }
 
+async function handleDeleteBooks(req, res) {
+  const { id } = req.params
+  try {
+    await Book.findByIdAndDelete(id);
+    res.status(204).send('succes')
+  } catch (e) {
+  res.status(200).send('server error')
```

Comment:
Good work on the lab so far!
- on line 74 we are passing in the err, but we aren't doing anything with it. So perhaps we can use this error by console logging the error inside the `catch` block and then send the appropriate error status code.
- on line 74 we we are sending a status of 200 as an error. Change this status code to an error code for server error which is 500 👍 
- on line 72, we can res.status(200).send('sucess') 👍